File: data/readable_format/fortran/file2141.f90
program square_test
implicit none

integer :: test_number, squared_number

! Set a test number
test_number = 4

! Calculate the square of the test number
squared_number = square(test_number)

! Print the result
print *, 'The square of', test_number, 'is', squared_number

contains

function square(x) result(squareResult)
    integer, intent(in) :: x
    integer :: squareResult

    squareResult = x * x
end function square

end program square_test

File: data/readable_format/cpp/file2141.cpp
#include <iostream>
using namespace std;

// Function to calculate the square of a number
int square(int x) {
    return x * x;
}

int main() {
    int test_number = 4;
    int squared_number;

    // Calculate the square
    squared_number = square(test_number);

    // Print the result
    cout << "The square of " << test_number << " is " << squared_number << endl;

    return 0;
}